trieste/acquisition/optimizer.py

```diff
@@ -523,13 +523,13 @@ def value_and_gradient(
 
             return cast("np.ndarray[Any, Any]", cache_y), cast("np.ndarray[Any, Any]", cache_dy_dx)
 
+        default_optimizer_args = dict(method="l-bfgs-b")
```

Comment:
Perhaps simpler like this

```suggestion
        optimizer_args = dict(method="l-bfgs-b", **(optimizer_args or {}))
```

and then just use `**optimizer_args` below?

Reply:
Actually that doesn't work as is. You have to wrap the defaults in a dict, otherwise you get multiple keyword args error when you try to override. So this would work instead:
`        optimizer_args = dict(dict(method="l-bfgs-b"), **(optimizer_args or {}))
`

Reply:
Updated that in the latest commits.